package.json

```diff
@@ -1,14 +1,18 @@
 {
   "name": "@openstax/highlighter",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "main": "dist/highlighter.js",
   "license": "MIT",
   "scripts": {
-    "typescript-watch": "tsc --noEmit --watch",
-    "webpack-watch": "webpack --config=webpack.config.js --mode=development --watch",
     "build": "webpack --config=webpack.config.js --mode=production",
     "jest-watch": "jest --watchAll",
-    "watch": "npm-run-all jest-watch webpack-watch typescript-watch"
+    "test": "jest --verbose --maxWorkers=2",
```

Comment:
Why are `maxWorkers` specifically set here?

Reply:
i copied it from tutor, i assume it makes jenkins turn into less of a spaceship but i can remove it